File: throwing-darts/throwing-darts.py
def score_throws(radiuses):
    s = sum(scora(i) for i in radiuses)
    return s if s!=10*len(radiuses) or not radiuses else 100+s

def scora(p):
    if p > 10:
        return 0
    return 10 if p < 5 else 5

File: throwing-darts/throwing-darts.js
function scoreThrows(radiuses){
  var s = radiuses.reduce((a,b)=>a+scora(b),0)
  return s!=10*radiuses.length || s==0 ? s:s+100
}


function scora(p){
    if(p > 10){return 0}
    return p < 5? 10 :5}